src/Symfony/Component/DependencyInjection/Compiler/PassConfig.php

```diff
@@ -57,7 +57,7 @@ public function __construct()
             new CheckDefinitionValidityPass(),
             new RegisterServiceSubscribersPass(),
             new ResolveNamedArgumentsPass(),
-            new AutowirePass(),
+            $autowirePass = new AutowirePass(),
```

Comment:
the new behavior should be opt-in to not break BC: add a constructor arg to AutowirePass?